README.md

```diff
@@ -1,10 +1,12 @@
 # LSP-css
-CSS, SCSS, LESS support for Sublime's LSP plugin.
 
-* Install [LSP](https://packagecontrol.io/packages/LSP) and `LSP-css` from Package Control. If you use Sass install [Sass](https://packagecontrol.io/packages/Sass).
-If you use Less install [LESS](https://packagecontrol.io/packages/LESS).
-* Restart Sublime.
+CSS, SCSS, LESS support for Sublime's LSP plugin using [vscode-css-languageserver](https://github.com/vscode-langservers/vscode-css-languageserver).
 
-### Configuration
+- Install [LSP](https://packagecontrol.io/packages/LSP) and `LSP-css` from Package Control.
```

Comment:
Sure. It's just because my linter doesn't like a `###` directly under a `#`.

![image](https://user-images.githubusercontent.com/6594915/83967962-3606ea00-a8f8-11ea-83a8-61628c928561.png)

Reply:
OK, pick something you want but we should be consistent. Maybe top level header should change to `##`?

Reply:
I just remain it `###`.